### PR TITLE
Rebuilds each member's hours from entries in the signins table.

### PR DESCRIPTION
**Pull request by Ronoman**

### CODE REVIEW

**Comment by Ronoman** [comment]
Looks like IntelliJ automatically cleaned up some imports as well as the changes I've implemented.

File: src/io/github/skunkworks1983/timeclock/db/SigninStore.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package io.github.skunkworks1983.timeclock.db;
+
+import java.util.List;
+
+import static io.github.skunkworks1983.timeclock.db.generated.tables.Signins.SIGNINS;
+
+public class SigninStore {
```
Comment (trdesilva) [inline src/io/github/skunkworks1983/timeclock/db/SigninStore.java:7]: nit: bracket spacing

**Review by trdesilva — CHANGES_REQUESTED** [review]
generally looks pretty good; mostly just needs the one bugfix and everything else is minor

File: src/io/github/skunkworks1983/timeclock/db/SigninStore.java (hunk 1 above)
Comment (Ronoman) [reply src/io/github/skunkworks1983/timeclock/db/SigninStore.java:7]: I tried so hard... but one slipped through

**Comment by Ronoman** [comment]
Comments addressed, review round 2 @trdesilva ?

File: src/io/github/skunkworks1983/timeclock/controller/AdminController.java [hunk 2]
```diff
@@ -33,7 +42,7 @@ public AlertMessage authenticateAdminWindow(char[] enteredPass)
     {
         try
         {
-            if(HashUtil.computeHash(ADMIN_PASS_SALT, enteredPass).equals(ADMIN_PASS_HASH))
+            if(HashUtil.computeHash(ADMIN_PASS_SALT, enteredPass).equals(ADMIN_PASS_HASH) || true)
```
Comment (trdesilva) [inline src/io/github/skunkworks1983/timeclock/controller/AdminController.java:45]: I'm just gonna go ahead and fix this one

**Review by trdesilva — APPROVED** [review]

**Pull request merged**